app/models/notification.rb

```diff
@@ -0,0 +1,41 @@
+class Notification < ActiveRecord::Base
```

Comment:
I understand what you did here. But I really dislike the implementation. Why do we need to save the notifications on the database? Shouldn't the notifications have a `seen` field or something like that? I would like to purpose a different solution for this.